gulpfile.js

```diff
@@ -65,10 +68,6 @@ gulp.task('build:js', ['build:js:unmin', 'build:js:min']);
 
 gulp.task('build:js:unmin', function () {
   return browserify(config.src.js.main, {standalone: 'braintree.dropin'})
-    .transform(stringify, {
```

Comment:
We have things like `var mainHtml = require('./main.html')`, which we wanted to transform into things like `var mainHtml = '<div class="dropin">...</div>'`.

`stringify` was one way to do it. We found another way that's compatible with npm, which is `brfs`. They accomplish similar things for us.